docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,215 @@
+- Keep focus on input when pressing list element ([#3092](https://github.com/mui/base-ui/pull/3092))
+- Allow <kbd>Esc</kbd> to bubble if `<Autocomplete.Empty>` is not used ([#2935](https://github.com/mui/base-ui/pull/2935))
+
+### Button
+
+- New `<Button>` component ([#2363](https://github.com/mui/base-ui/pull/2363))
+
+### Collapsible
+
+- Fix `starting-style` state ([#2985](https://github.com/mui/base-ui/pull/2985))
+
+### Combobox
+
+- Take into account `isItemEqualToValue` when selecting an option in multiple mode ([#2893](https://github.com/mui/base-ui/pull/2893))
+- Move `CompositeList` to `List` component to make `Input` work with composites ([#2883](https://github.com/mui/base-ui/pull/2883))
+- Fix `onValueChange` type inference when `value` is unspecified ([#2897](https://github.com/mui/base-ui/pull/2897))
+- Fix `required` form submission with multiple values ([#2925](https://github.com/mui/base-ui/pull/2925))
+- Fix <kbd>Home</kbd>/<kbd>End</kbd> Input scroll in Chrome/Safari ([#2928](https://github.com/mui/base-ui/pull/2928))
+- Prevent blocking filtering while composing text on Android ([#2944](https://github.com/mui/base-ui/pull/2944))
+- Add empty state to `List.State` ([#2934](https://github.com/mui/base-ui/pull/2934))
+- Fix `initialFocus` as function being called on close ([#2949](https://github.com/mui/base-ui/pull/2949))
+- Add `role="combobox"` to `<Combobox.Trigger>` if `<Combobox.Input>` is inside Popup ([#2973](https://github.com/mui/base-ui/pull/2973))
+- Fix Field control ref when input is inside popup ([#2971](https://github.com/mui/base-ui/pull/2971))
+- Fix stale `onItemHighlighted` data when filtering with `autoHighlight` ([#2829](https://github.com/mui/base-ui/pull/2829))
+- Add empty and side styling attributes on `<Combobox.Input>` ([#2926](https://github.com/mui/base-ui/pull/2926))
+- Fix `<Combobox.Value>` component return type for React 17 ([#3050](https://github.com/mui/base-ui/pull/3050))
+- Fix input value derivation on `value` and `items` prop updates ([#3067](https://github.com/mui/base-ui/pull/3067))
+- Support `autoHighlight: "always"`, and add `keepHighlight`, `highlightItemOnHover` props ([#2976](https://github.com/mui/base-ui/pull/2976))
+- Keep focus on input when pressing list element ([#3092](https://github.com/mui/base-ui/pull/3092))
+- Fix support of dialog + combobox pattern ([#3049](https://github.com/mui/base-ui/pull/3049))
+- Support drag-to-select ([#3167](https://github.com/mui/base-ui/pull/3167))
+- Allow <kbd>Esc</kbd> to bubble if `<Combobox.Empty>` is not used ([#2935](https://github.com/mui/base-ui/pull/2935))
+- Fix stuck filtering with differing stringifiers ([#3201](https://github.com/mui/base-ui/pull/3201))
+
+### Context Menu
+
+- Add `open` state to `<ContextMenu.Trigger>` ([#3195](https://github.com/mui/base-ui/pull/3195))
+- Fix ignored `anchor` prop on `<ContextMenu.Positioner>` ([#3202](https://github.com/mui/base-ui/pull/3202))
+
+### Dialog
```

Comment:
`Dialog` and `AlertDialog` are missing dot points for the new `<Dialog.Viewport>` / `<AlertDialog.Viewport>` part